sqlglot/dialects/singlestore.py

```diff
@@ -1151,4 +1160,37 @@ class Generator(generator.Generator):
             "YES",
             "ZEROFILL",
             "ZONE"
-        }
\ No newline at end of file
+        }
+
+        def all_sql(self, expression: exp.All) -> str:
+            self.unsupported(
+                "ALL subquery predicate is not supported in SingleStore")
+            return super().all_sql(expression)
+
+        def any_sql(self, expression: exp.Any) -> str:
+            self.unsupported(
+                "ANY subquery predicate is not supported in SingleStore")
+            return super().any_sql(expression)
+
+        def glob_sql(self, expression: exp.Glob) -> str:
+            self.unsupported("GLOB predicate is not supported in SingleStore")
+            return super().glob_sql(expression)
+
+        def ilike_sql(self, expression: exp.ILike) -> str:
```

Comment:
Is it not possible to override this? Also keep in mind that `LIKE` in SingleStore acts as `ILIKE` in Postgres, `LIKE BINARY` as `ILIKE`.